src/schema/updates/manifest.json

```diff
@@ -41,10 +41,12 @@
                 "properties": {
                   "background": {
                     "anyOf": [
-                      {},
-                      {},
                       {
-                        "min_manifest_version": 3
+                        "properties": {
+                          "service_worker": {
+                            "min_manifest_version": 3
+                          }
+                        }
```

Comment:
note: this change on the `src/schema/updates/manifest.json` was needed to ensure the `min_manifest_version`: 3 will still get applied to the `service_worker` manifest json property schema (otherwiser it was being applied to the entire `background` JSONSchema definition, which was part of the reasons for the test failures being hit by the initial version of this patch).